src/scene/scenegraphnode.cpp

```diff
@@ -309,6 +326,15 @@ namespace {
             // scene graph node. This is most useful to trim collective lists of nodes and
             // not display, for example, barycenters
             std::optional<bool> hidden;
+
+            // If this value is specified, the scene graph node will be ordered in
+            // relation to its neighbors in the GUI based on this value, so that nodes
+            // with a higher value appear later in the list. Scene graph noeds with the
+            // same value will be sorted alphabetically.
+            //
+            // The nodes without a given value will be placed at the top at the list and
+            // sorted alphabetically.
```

Comment:
I thought we wanted to move the items without to the bottom of the list?

Reply:
I did, just forgot to change this comment  :) Thanks